src/test/java/TestQueryChecker.java

```diff
@@ -268,21 +286,24 @@ public void testValidQueryChecker() throws Exception {
 
   @Test
   public void testInvalidQueryChecker() throws Exception {
-    HashMap<NNAConstants.SET, HashMap<String, HashMap<String, ArrayList<EnumSet<NNAConstants.FILTER>>>>> setSumTypeFilterConfig = getInvalidCombinations();
+    HashMap<
```

Comment:
is this really what checkstyle suggests ? looks untidy to me.

Reply:
Oh wow good catch. Yeah that looks really untidy.

Reply:
The bigger issue here is the crazy level of nesting in this container. Creating some classes to wrap this data structure might be the next step, but I'm not super concerned about the untidiness above per se.